docs/docs-beta/docs/dagster-plus/insights.md

```diff
@@ -1,5 +1,157 @@
 ---
 title: "Insights"
+description: "Visiblity into historical usage, cost, and metadata."
 ---
 
-# Dagster+ Insights
\ No newline at end of file
+# Dagster+ Insights
+
+Using Dagster+ Insights, you can gain visibility into historical usage and cost metrics such as Dagster+ run duration, credit usage, and failures.
+Visualizations are built right into the Dagster UI, allowing you to explore metrics from Dagster as well as downstream systems (like Google BigQuery) all in one place.
+
+### With Insights, you can
```

Comment:
I definitely diverged from the "in this guide, you will learn" format here -- open to suggestions.

Reply:
I'm on the fence, but I don't think it's a blocker.